Documentation/source/changelog.rst

```diff
@@ -6,7 +6,8 @@ Changelog
 ==================
 
 - Adapted UI for iPad (landscape orientation is now supported on iPad)
-- Fixed issue on preview screen, when there is no orientation (device on flat position) 
+- Fixed issue on preview screen, when there is no orientation (device on flat position)
+- Removed MotionManager class which led to UX issues. 
```

Comment:
The MotionManager is implementation detail our clients don't know about. We should communicate this in the changelog in a way that reflects the impact the clients can notice, for ex.: `Disabled automatic picture rotation on iPhones. Previously the picture was rotated automatically if it was taken while the device was held in landscape. Now the picture is always shown in portrait in the Review Screen.`